packages/block-directory/package.json

```diff
@@ -29,9 +29,11 @@
 		"@wordpress/blocks": "file:../blocks",
 		"@wordpress/components": "file:../components",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "file:../core-data",
```

Comment:
Nice, this is why we do it in the first place - to ensure all deps are properly listed.